src/Logger.cpp

```diff
@@ -40,39 +41,95 @@ void Logger::Start(time_t time, const StringRef& filename) noexcept
 			lastFlushFileSize = logFile.Length();
 			logFile.Seek(lastFlushFileSize);
 			logFileName.copy(filename.c_str());
-			InternalLogMessage(time, "Event logging started\n");
+			String<40> startMessage;
```

Comment:
Don't use String<40>, use one of the predefined StringLength values, so that we get fewer instantiations of the String template

Reply:
Used `StringLength50` instead.